hydrolib/core/io/ini/util.py

```diff
@@ -462,3 +462,28 @@ def is_valid_coordinates_with_num_coordinates_specification() -> bool:
         raise ValueError(error)
 
     return root_validator(allow_reuse=True)(validate_location_specification)
+
+
+def get_key_renaming_root_validator(keys_to_rename: Dict[str, List[str]]):
+    """
+    Gets a root validator that renames the provided keys to support backwards compatibility.
+
+    Args:
+            keys_to_rename Dict[str, List[str]]: Dictionary of keys and a list of old keys that
+            should be converted to the current key.
+    """
+
+    def rename_keys(cls, values: Dict) -> Dict:
+        for current_keyword, old_keywords in keys_to_rename.items():
+            if values.get(current_keyword) is not None:
```

Comment:
```suggestion
            if current_keyword in values:
```

Just another way to do it, but should not be changed.

Reply:
I like your way better. It's a little cleaner. I will change it!